app/models/miq_template/operations.rb

```diff
@@ -1,4 +1,10 @@
 module MiqTemplate::Operations
+  extend ActiveSupport::Concern
+
+  included do
+    supports_not :reboot_guest, :reason => _("Reboot Guest Operation is not available for Images/Templates.")
```

Comment:
Here you are right :smile: 
@Fryguy and once you merge https://github.com/ManageIQ/manageiq/pull/10944 this would even raise some gettext exception :smile: